acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/TransactionSignerParamsSupplier.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.dsl.signer;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import tech.pegasys.ethsigner.tests.dsl.Accounts;
+import tech.pegasys.ethsigner.tests.hashicorpVault.HashicorpVaultDocker;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import com.google.common.io.Resources;
+
+public class TransactionSignerParamsSupplier {
```

Comment:
I'm not convinced this is the best direction, using a single class to manage all the types of launch parameters. Didn't we talk about interface and separate implementations for the file and vault?

Reply:
We did, but before we had that discussion we said that we want the signer part of the params to be dealt with in a class. If we decide to go with the "runner" then I think we should do that outside this ticket.

Reply:
I'm inclined to disagree 🤔  ... I'd interpret it as being part of the refactoring required to introduce this feature.